frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -215,10 +215,205 @@ describe('Monitoring: YAML', () => {
     expect(monitoringView.alertManagerYamlForm.isPresent()).toBe(true);
   });
 
-  it('saves alert-manager.yaml', async () => {
+  it('saves Alertmanager YAML', async () => {
     expect(monitoringView.successAlert.isPresent()).toBe(false);
     await monitoringView.saveButton.click();
     await crudView.isLoaded();
     expect(monitoringView.successAlert.isPresent()).toBe(true);
   });
 });
+
+describe('Alertmanager: Configuration', () => {
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('displays the Alermanager Configuration Overview page', async () => {
+    await sidenavView.clickNavLink(['Administration', 'Cluster Settings']);
+    await crudView.isLoaded();
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+    await monitoringView.wait(
+      until.elementToBeClickable(monitoringView.firstListLinkById('alertmanager')),
+    );
+    expect(monitoringView.firstListLinkById('alertmanager').getText()).toContain('Alertmanager');
+    await monitoringView.firstListLinkById('alertmanager').click();
+    await crudView.isLoaded();
+    expect(monitoringView.alertRoutingHeader.getText()).toContain('Alert Routing');
+  });
+
+  it('launches Alert Routing modal, edits and saves correctly', async () => {
+    await crudView.isLoaded();
+    expect(monitoringView.alertRoutingEditButton.isPresent()).toBe(true);
+    await monitoringView.alertRoutingEditButton.click();
+    await crudView.isLoaded();
+
+    expect(monitoringView.firstListLinkById('input-group-by').isPresent()).toBe(true);
```

Comment:
We shouldn't be using a `firstLinkById` helper to get items that aren't links. Maybe we need a generic `data-test-id` helper

Reply:
fixed